copylot/assemblies/photom/gui/windows.py

```diff
@@ -136,7 +139,7 @@ def initialize_UI(self):
             self.photom_window_pos[0],
             self.photom_window_pos[1],
             self.photom_window_size_x,
-            self.photom_window_size_x,
+            1000,
```

Comment:
let's not hardcode this variable. If needed, change the `photom_gui.py` file that executes this PhotomApp window.